### DISCUSSION TITLE
Tunnel unpredictable goes down every hour.: trap not foud

### DISCUSSION THREAD

**Discussion by misch42** [strongswan/strongswan#2306] [General]
Hello,

we have a VPN tunnel to a Cisco ASA. The connection breaks sometimes when renegotiating the CHILD_SA. The only suspicious entry in the logs are:

charon-systemd[666244]: 12[KNL] creating acquire job for policy a.a.a.a/32[tcp/50546] === b.b.b.b/32[tcp/server] with reqid {4}
charon-systemd[666244]: 12[CFG] trap not found, unable to acquire reqid 4

The config is:
(...)
  child {
    local_ts = ...
    remote_ts = ...
    rekey_time = 3600
    start_action = start
    close_action = trap
}

Any ideas what might go wrong?

Michael.

**Comment by tobiasbrunner** [comment]
> Any ideas what might go wrong?

Many ideas but without log and the strongSwan version we can't tell what's actually going on. Also, what's the idea behind using `start_action = start` and `close_action = trap`?
